rs/nns/governance/canister/canister.rs

```diff
@@ -385,6 +385,7 @@ fn canister_post_upgrade() {
         restored_state.neurons.len(),
         restored_state.xdr_conversion_rate,
     );
+    let restored_state = migrate_neurons_fund_audit_info(restored_state);
```

Comment:
Is this the right place to run the migration? Cc @max-dfinity

Reply:
Yes, assuming the migration isn't very expensive.

Reply:
🙏